devstack_vm/tests/include-tests.txt

```diff
@@ -0,0 +1,2 @@
+tempest.
+neutron_fwaas.
```

Comment:
not sure we want to run eeeevery tempest test for this. A subset that specifically does something on the compute nodes should sufice:


tempest.api.compute.admin
tempest.api.compute.servers
tempest.api.compute.test_live_block_migration_negative
tempest.api.compute.volumes
tempest.scenario

and maybe:

tempest.api.network